packages/reactive-element/src/reactive-element.ts

```diff
@@ -608,35 +609,21 @@ export abstract class ReactiveElement
    */
   static styles?: CSSResultGroup;
 
-  /**
-   * The set of properties defined by this class that caused an accessor to be
-   * added during `createProperty`.
-   *
-   * This field is only used in dev mode. It should be eliminated in prod
-   * builds.
-   *
-   * @nocollapse
-   */
-  private static __reactivePropertyKeys?: Set<PropertyKey>;
```

Comment:
Removing this caused a failing DEV_MODE test case in GitHub Actions.

Reply:
yeah... working on it

Reply:
fixed